### ISSUE TITLE
Dead area right-side of scrollbar

### ISSUE THREAD

**Issue by tushar-singh** [Komodo/KomodoEdit#131]
There's a 1px wide dead area right side of scrollbar. As a result, you can't use scrollbar if you drag your mouse all the way to right side.
It appears to be a 1px border around the editing area which contains the scrollbar. Thus, 1px wide no-scroll area

[Screenshot](https://cloud.githubusercontent.com/assets/1689818/7020482/f5e13dd2-dd36-11e4-953f-70690b6e00bb.png) (easily visible when seen on black background-http://imgur.com/egkyR8D) 

OS: windows 8.1 x64
Komodo Edit, version 9.0.0, build 15707, platform win32-x86.

**Comment by Naatan** [comment]
@cgchoffman are you seeing this on Windows?

**Comment by cgchoffman** [comment]
Nope.  Might be a Windows 8 thing.  Checked in Edit and IDE. I can't check Win8 when i get home this evening.

**Comment by dsuch** [comment]
Hello,

this is not a Windows-only thing. I can confirm it on Ubuntu 14.04 under:

```
Komodo Edit, version 9.0.1, build 15745, platform linux-x86_64.
Built on Fri Apr 24 08:00:26 2015.
```

Can you please tell me how it works for you?

Here when you drag the mouse to the right side you cannot use the scrollbar, you need to place the cursor a pixel to the left.

This is very distracting.

Thanks a lot.

**Comment by dsuch** [comment]
Following up with details - this only happens if the right pane is in use.

If only the editor window is active, no right pane, it all works as expected.

But as soon as one opens the right pane, its scrollbar (that of the pane) is a different one and it has that additional pixel worth of dead area.

**Comment by cgchoffman** [comment]
The reason I didn't notice this on windows 7 is that on Win7 the app window has a border that gives it padding relative to everything around it.  In Win8 that padding doesn't exist so when lined up with other aspects of Windows OS (eg. the edge of the taskbar) there is a one pixel discrepancy.  I don't notice any functional difference.

**Comment by Naatan** [comment]
Looking closer .. isn't that just the window border?

**Comment by tushar-singh** [comment]
Window border would be around the entire window, this border is only on right side of scrollbar

**Comment by cgchoffman** [comment]
What makes you think it's not? The edge of the scroll bar lines up at the top with the rest of the window.  I think Nathanr is right.  Is this actually causing Komodo to malfunction?  I don't see how this prevents one from scrolling.

**Comment by tushar-singh** [comment]
From my first post "As a result, you can't use scrollbar if you drag your mouse all the way to right side."

Generally when i want to use scrollbar to scroll, i drag my mouse all the way to right edge of screen. This works for every other app but in Komodo it doesn't work and i have to aim for the narrow scrollbar

**Comment by cgchoffman** [comment]
Talk about an "edge case".  Kidding!  There are higher priority issues for us to deal though.  We'll get to this when we can.  

> Following up with details - this only happens if the right pane is in use.

There is an easy work around.  To enable/disable the right pane quickly you can use Ctrl + L (show hide toolbox, Show right pane isn't keybound by default).

**Comment by tushar-singh** [comment]
I agree its not a critical issue, just a slight UX problem.
For me it happens with or without right pane. I only use left pane and i tried to disable that but issue still persists. Anyway, hopefully this gets fixed someday
Thanks for your time

**Comment by cgchoffman** [comment]
Thank you for understanding and your patience.

**Comment by Naatan** [comment]
I believe this should be fixed in 9.2
